tasks/agent5-linux.yml

```diff
@@ -26,6 +26,21 @@
     enabled: no
   when: not datadog_skip_running_check and not datadog_enabled
 
+- name: Register all checks directories present in datadog
+  find:
+    paths: /etc/dd-agent/conf.d/
+    pattern: "*.yaml"
+    file_type: directory
+  register: datadog_conf_directories
+  when: datadog_disable_untracked_checks
+
+- name: Delete checks not present in datadog_tracked_checks
+  file:
+    path: "/etc/dd-agent/conf.d/{{ item }}.yaml"
```

Comment:
This will remove any user-defined configuration of the checks, but in the case of core checks (disk, memory, etc.), they will still be running with the default configuration, as the default config file (`{{ item }}.yaml.default`) will still be present.
Is this expected? Or should the default files also be removed?

(Same question for Agent 6, with the `conf.d/{{ item }}.d/conf.yaml.default` files)

Reply:
While it would be useful to keep them as a base case, I think it'll make things a lot complexer when people start modifying the defaults - and there are likely use cases where you want to remove the defaults anyway.

Maybe an additional config option (`datadog_disable_default_checks`)?

Reply:
Hey ! I followed @lesteenman idea by adding a `datadog_disable_default_checks` option (true or false). So it will remove all defaults checks or not.
If you want to go further with the idea you can pick it up to choose which one you will prefer to remove (list).